packages/js/src/lib/person.ts

```diff
@@ -117,30 +78,57 @@ export const hasAttributeKey = (key: string): boolean => {
 export const setPersonUserId = async (
```

Comment:
we should remove the `setPersonUserId` method as we don't have the `formbricks.setUserId()` method (since we set this in init)